lib/final_destination.rb

```diff
@@ -8,6 +8,8 @@
 
 # Determine the final endpoint for a Web URI, following redirects
 class FinalDestination
+  MAX_TIME = 10
```

Comment:
Would recommend a little more descriptive name for these, `MAX_REQUEST_TIME_SECONDS` and `MAX_REQUEST_SIZE_BYTES`